INSTALL.md

```diff
@@ -0,0 +1,38 @@
+# AEGeAn Installation and Setup
+
+## Installation as a singularity container [![https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg](https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg)](https://singularity-hub.org/collections/3301)
```

Comment:
I'm a big fan of the container approach! My only concern is that the AEGeAn documentation doesn't become fragmented. I've been using ReadTheDocs to host [AEGeAn's documentation](https://aegean.readthedocs.io/en/stable/) for a while—it detects when the master branch has been updated or a new release has been published, and rebuilds the documentation from the `docs/` directory automatically. We don't want redundant or conflicting installation procedures in different places.

I think the installation instructions you've provided here are a great improvement over the install procedures we've used in the past. So the question is whether it makes sense to move these new instructions into the `docs/` directory? Or move the other contents of the `docs/` directory  somewhere else?